src/data/linux-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const linuxQuiz = [
+  {
+    Question: "Question",
+    Answer: "A",
+    Distractor1: "B",
+    Distractor2: "C",
+    Distractor3: "D",
+    Explanation: "Explanation",
+    Link: "link"
+  },
```

Comment:
You will need to provide an actual question and set of answers here otherwise we will have to close this PR

Please update this 👍

Reply:
I have updated the file and corrected the mistake. Please take a review